nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -129,6 +129,7 @@ define([
                 bus: runtime.bus(),
             }),
             typesToFiles = setupFileData(options.importData),
+            { fileTypesDisplay } = BulkImportUtil.generateFileTypeMappings(typesToFiles),
```

Comment:
this gets used by several tabs, so calculate it here instead of doing it separately several times